src/main/java/pl/allegrotech/productsshop/domain/Product.java

```diff
@@ -8,11 +9,13 @@
   private final String id;
   private final String name;
   private final LocalDateTime createdAt;
+  private final BigDecimal price;
 
-  Product(String id, String name, LocalDateTime createdAt) {
+  Product(String id, String name, LocalDateTime createdAt, BigDecimal price) {
```

Comment:
Można by dodać też walutę (np Currency, albo Joda Money) bo teraz nie wiadomo w jakiej walucie zapisujemy cenę, a w konwerterze na sztywno podajemy PLN.

Reply:
Myślałem o tym, ale zdecydowałem się to olać żeby nie komplikować całości. Zakładam po prostu, że ceny są w PLNach i tyle.